sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 - `GetDataLakePathPropertiesResult::AccessTierInferred` was renamed to `IsAccessTierInferred`.
 - `HttpHeaders` of `DownloadDataLakeFileResult` and `DownloadDataLakeFileToResult` was moved into `Details`, to align with Blob service.
 - Removed `BreakDataLakeLeaseResult::LeaseTime`.
+- Renamed APIs for modifying access list recursively. Used to be with pattern `AccessControlRecursiveList`, now is with pattern `AccessControlListRecursive`.
+- Refined options for `ScheduleDeletion`, to be consistent with other APIs.
+- Renamed `ContentLength` in `PathItem` to `FileSize`.
+- In `PathSetAccessControlRecursiveResult`, `DirectoriesSuccessful` is renamed to `NumberOfSuccessfulDirectories`, `FilesSuccessful` is renamed to `NumberOfSuccessfulFiles`, `FailureCount` is renamed to `NumberOfFailures`.
```

Comment:
Just for the record, it's not that I prefer the name patter `NameOfXxx`, it's just I think we should keep consistent.

Reply:
Those names are defined in swagger, meaning service is not making them consistent. However, I don't have a preference so I made the change.